pulp_rpm/app/constants.py

```diff
@@ -1,5 +1,16 @@
 from types import SimpleNamespace
 
+
+RPM_PLUGIN_TYPES = SimpleNamespace(
+    PACKAGE='rpm.package',
```

Comment:
This is the type of hardcode I hoped to avoid. Is it bad to do:
`PACKAGE = models.Package.TYPE` ? Is it because of prepending with app_label that we need to hardcode the type?

Reply:
> Is it because of prepending with app_label that we need to hardcode the type?

Yup, because the value saved in _type is not the same as models.Package.TYPE

Reply:
Likely not a popular opinion:
Maybe we can use existing names and if we don't go fancy with simple namespaces, it can be done [arguably] easier.
E.g. for the available types use `Package.TYPE`,
for the type to look for in db add app_label to it.
```
content_filter_choice = (Package, UpdateRecord)
supported_types = [model.TYPE for model in content_filter_choice]  # to validate users' input
final_types = ['{app_label}.{type}'.format(app_label=model._meta.app_label, type=model.TYPE)
               for model in user_specified_types]
```

Reply:
We can do this as part of the refactor if you want, I'm not opposed.  If others are fine with it also than it works for me.

Reply:
We can do this as part of the refactor if you want, I'm not opposed.  If others are fine with it also than it works for me.